velox/functions/sparksql/Hash.cpp

```diff
@@ -95,13 +327,13 @@ void applyWithType(
 
 class Murmur3Hash final {
```

Comment:
are there any benefits on defining the SeedType and ReturnType traits somewhere and not directly in this class? e.g.

```
class Murmur3Hash final {
  using SeedType = int32_t;
  using ReturnType = int32_t;
  ...
```